packages/plugin-odoo/src/Odoo.ts

```diff
@@ -135,4 +267,149 @@ export class Odoo<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
             ['OdooSaveButton', ['actionables']],
         ],
     };
+
+    commands = {
+        setOdooImagePadding: {
+            handler: this.setImagePadding,
+        },
+        setOdooImageWidth: {
+            handler: this.setOdooImageWidth,
+        },
+        setOdooImageClass: {
+            handler: this.setOdooImageClass,
+        },
```

Comment:
Est-ce que c'est vraiment Odoo ? Je ne dis pas de sortir ces commandes du plugin Odoo, mais est-ce qu'elle doivent avoir "Odoo" dans leur nom ? Je veux dire, outre le fait qu'elles ne sont utilisées que dans Odoo, elles ne marcheraient carrément pas en dehors d'Odoo ? J'ai l'impression que si. D'allleurs le nommage est inconsistant parce que la commande `setOdooImagePadding` devient `setOdooImagePadding` et `setOdooImageWidth` prends des `SetImageWidthParams` donc je pense que dans le fond on est d'accord et qu'on pourrait retirer "Odoo" des noms de commandes et de fonctions.

Reply:
You are rigth. I changed it. :+1: